cli/src/clap_app.rs

```diff
@@ -166,6 +166,22 @@ pub fn get_clap_app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> A
                                 .required(true),
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("default")
```

Comment:
this name is too generic and can be eliminated if the proposal to move from static override path to config field is adopted